easybuild/tools/modules.py

```diff
@@ -555,13 +556,35 @@ def mod_exists_via_show(mod_name):
             # - Check first non-whitespace line for something that looks like an absolute path terminated by a colon
             mod_exists_regex = r'\s*/.+:\s*'
             for line in stderr.split('\n'):
+                # skip whitespace lines
+                self.log.debug("Checking line '%s' to determine whether %s exists...", line, mod_name)
```

Comment:
As you now already output on every decision IMO this line can go. If you really want to keep it move it above the comment please

Reply:
I prefer keeping it, so I moved it